src/xray_build.props

```diff
@@ -49,7 +49,7 @@
   <!-- Common debug settings -->
   <PropertyGroup Condition="$(Configuration.Contains('Debug'))" Label="Configuration">
     <UseDebugLibraries>true</UseDebugLibraries>
-	<EnableASAN>true</EnableASAN>
+    <EnableASAN>false</EnableASAN>
```

Comment:
Just remove the line, let it be the Microsoft's default :)
(which is false anyway)

Reply:
Yup that's even easier